src/plugins/home/public/application/components/change_home_route/change_home_route.tsx

```diff
@@ -32,20 +32,26 @@ export const ChangeHomeRoute: FunctionComponent<Props> = ({ defaultRoute }) => {
   const changeDefaultRoute = () => uiSettings.set('defaultRoute', defaultRoute);
 
   return (
-    <EuiText>
-      <p>
-        <FormattedMessage
-          id="home.changeHomeRouteText"
-          defaultMessage="Would you prefer to have an alternate home page for this Elastic space? "
-        />
-        <EuiButtonEmpty iconType="home" onClick={changeDefaultRoute}>
+    <EuiFlexGroup className="homPage__footer" alignItems="center" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <EuiText size="s" color="subdued">
+          <p>
+            <FormattedMessage
+              id="home.changeHomeRouteText"
+              defaultMessage="Would you prefer to have an alternate home page for this Elastic space? "
+            />
+          </p>
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiButtonEmpty size="xs" iconType="home" onClick={changeDefaultRoute}>
           <FormattedMessage
             id="home.changeHomeRouteLink"
             defaultMessage="Change your home page route"
           />
         </EuiButtonEmpty>
-      </p>
-    </EuiText>
+      </EuiFlexItem>
+    </EuiFlexGroup>
```

Comment:
I'm wondering if it would be better to not utilize `EuiFlexGroup` in this situation, as I fear it has the potential to yield an odd layout. For example, assuming we had a long string of text that breaks a line (in the first `EuiFlexItem`), this would cause the button (in the second `EuiFlexItem`) to stay positioned to the right and vertically center itself. I'm not sure that's desirable in this situation.

![image](https://user-images.githubusercontent.com/3884767/87972787-80ea5480-ca95-11ea-8fd3-c5d683edc273.png)

Would it be better to have the button appear inline with the paragraph text, so that if/when the line is broken, the button also moves to the second line (thus avoiding the above odd layout)?

Reply:
Probably so and its worth tinkering with it. The reason I swapped it to use the flex components was to get the text and empty button to align vertically. It was visually misaligned when the button was inlined in the paragraph.